### DISCUSSION TITLE
Support token introspection endpoint

### DISCUSSION THREAD

**Discussion by Samrose-Ahmed** [jsforce/jsforce#1445] [Ideas]
Would be convenient to have a method to check token status.
Right now I'm just making call to [Salesforce token introspection endpoint](https://help.salesforce.com/articleView?id=remoteaccess_oidc_token_introspection.htm&type=5).

**Comment by tcd** [comment]
@Samrose-Ahmed Link doesn't seem to be working

**Comment by Samrose-Ahmed** [comment]
Weird, seems the link has gone down. Its also documented [here](https://help.salesforce.com/articleView?id=remoteaccess_oidc_dynamic_client_reg_flow.htm&type=5) under section 'Request Token Introspection'

**Comment by AllanOricil** [comment]
openid-client helps with it

https://github.com/jsforce/jsforce/discussions/1744
